lib/trento_web/controllers/v1/host_json.ex

```diff
@@ -60,7 +62,7 @@ defmodule TrentoWeb.V1.HostJSON do
   def saptune_status_updated(%{
         host: %{id: id, saptune_status: status, hostname: hostname}
       }),
-      do: %{id: id, status: status, hostname: hostname}
+      do: %{id: id, status: StructHelper.to_atomized_map(status), hostname: hostname}
```

Comment:
This concerns me. Why we are converting something to atom-keys when actually, phoenix will reconvert this back to json/string?

Reply:
Because of what you mentioned
> I guess that all of this is caused for adding new things to the factory. All saptune status data coming from the database should already have atomized keys.

Reply:
here we go https://github.com/trento-project/web/pull/3535/commits/eeda9c4f095e372cfc02bd2cd5d6278b2b2a1e34